LICENSE.mustache

```diff
@@ -31,10 +31,10 @@ infringe it by using this software as of my latest contribution.
    part of a larger program.
 
 To release source code, publish the preferred form for making
-changes through a widely used, freely accessible source code
-distribution system, and license it on terms at least as
-permissive as this license. If source code has already been
-released, you do not have to release it again.
+changes through a freely accessible source code distribution
+system indexed by public search engines, and license it on terms
```

Comment:
In the end, this feels too technology-dependent. I could back up, and require a distro system that is "freely accessible and easy to search", but that's probably a _worse_ proxy for how developers will actually find code than popularity.